src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -182,6 +182,16 @@ to this directory.",
             true,
           )),
       )
+        .subcommand(
+          SubCommand::with_name("gc")
+              .about("Garbage collect the on-disk store. Note that after running this command, any processes with an open store (e.g. a pantsd) may need to re-initialize their store.")
```

Comment:
This should only be the case if the leasing mechanism fails, right?

EDIT: Having read further, I guess the `compact` strategy will also cause this, but `fast` should not?